Tasks/ServiceFabricPowerShell/task.json

```diff
@@ -0,0 +1,90 @@
+{
+    "id": "816979F7-C273-4347-9A55-845B721D82CB",
+    "name": "ServiceFabricPowerShell",
+    "friendlyName": "Service Fabric PowerShell",
+    "description": "Run a PowerShell script within the context of an Azure Service Fabric cluster connection.",
+    "helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkID=841538)",
```

Comment:
Make sure to get a new FWLink ID created for this and have it linked to the task documentation page.